server/database/ModelDefinitions.js

```
@@ -0,0 +1,397 @@
+/**
+ * Model Definitions
+ * Database-agnostic model schemas
+ */
+
+// Import data types based on database type
+const getDataTypes = (databaseType) => {
+	switch (databaseType) {
+		case 'sequelize':
+			const { DataTypes } = require('sequelize');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)